backend/src/packages/files/libs/helpers/create-invalid-file-error-message.helper.ts

```diff
@@ -0,0 +1,44 @@
+import { type MultipartFile } from '@fastify/multipart';
+
+import { type ValueOf } from '~/libs/types/types.js';
+
+import { FilesValidationError } from '../enums/enums.js';
+import { type FileInputConfig } from '../types/types.js';
+import { filesize } from './helpers.js';
+
+const createInvalidFileErrorMessage = (
+  errorType: ValueOf<typeof FilesValidationError>,
+  fileInputConfig: Required<FileInputConfig>,
+  file?: MultipartFile,
+): string => {
+  switch (errorType) {
+    case FilesValidationError.FILE_TOO_BIG: {
+      const problemFile = file as MultipartFile;
+
+      return `File '${problemFile.filename}': size must be less than ${filesize(
+        fileInputConfig.maxSize,
+      )}`;
+    }
+    case FilesValidationError.TOO_MANY_FILES: {
+      return `There can be no more than ${fileInputConfig.maxFiles} file${
+        fileInputConfig.maxFiles > 1 ? 's' : ''
```

Comment:
maybe let's create a separate helper in shared and reuse in all places where we need to decide whether to write with 's' or without